runtime/CMakeLists.txt

```diff
@@ -7,21 +7,20 @@
 #===------------------------------------------------------------------------===#
 
 add_library(FortranRuntime
-  ISO_Fortran_binding.cc
-  derived-type.cc
-  descriptor.cc
-  file.cc
-  format.cc
-  io-api.cc
-  io-error.cc
-  io-stmt.cc
-  main.cc
-  memory.cc
-  stop.cc
-  terminator.cc
-  tools.cc
-  transformational.cc
-  type-code.cc
+  ISO_Fortran_binding.cpp
+  derived-type.cpp
+  descriptor.cpp
+  format.cpp
```

Comment:
You deleted file.cc.

Reply:
Thank you.  I had been unable to find the extra deletion myself.  I believe [https://github.com/flang-compiler/f18/pull/953/commits/dafa1a95db8c02014bafa40f8c0e89b3610cebed](url) fixes this.